app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -89,6 +89,30 @@ describe("Summary", () => {
     await screen.findByTestId("pill-enrolling-complete");
   });
 
+  it("renders the end experiment button if the experiment is live and idle", async () => {
+    render(
+      <Subject
+        props={{
+          status: NimbusExperimentStatusEnum.LIVE,
+          publishStatus: NimbusExperimentPublishStatusEnum.IDLE,
+        }}
+      />,
+    );
+    await screen.findByText("End Experiment");
```

Comment:
You don't need to use expect to be in the document here?  Because of the await it'll throw if it can't find it?

Reply:
Oh I see this is the pattern used in all the other tests.  Yep looks good 👍